share/revng/test/tests/analysis/SimplifySwitch/x86_64/simple-switch.S

```diff
@@ -0,0 +1,100 @@
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+    .intel_syntax noprefix
+    .type myfunc, @function
+    .globl myfunc
+myfunc:
+    mov    rax, QWORD PTR [rax]
```

Comment:
1. Use `rdi`.
2. This function does not make much sense to me, there's a lot of unreachable code.
   Why do you create so much code but then just use 5?
   If you just use 5, why not conflate this test into `simple-switch.S`.

Reply:
We will use the existing `x86_64/switch-jump-table.S` test. Thanks.